### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by tonidew**
This is my HTML & CSS quiz. It is ready to be reviewed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title<title> 
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+
+<div class="Container">
```
Comment (C-Stein) [inline index.html]: This div will need to be indented inside the body

File: index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      <article class="card">
+        <header class="mainHeader"><h1>Yellow Dunk Cards</h1></header>
+        
+      <!-- The first section should contain three child block elements.*/ -->
+      <!-- /*The first element contains the product image. -->
+      <!-- The second element contains the product description. -->
+      <!-- The third element contains the product availability (e.g. "Available" or "Not Available") -->
+
+        <section>
+          <img src="http://www.clipartbest.com/cliparts/9ai/qEb/9aiqEb9cM.jpeg" class="productImage" alt="Yellow Square">
```
Comment (C-Stein) [inline index.html]: img elements aren't exactly block level elements, you  need to wrap this image in another tag to meet the requirements

File: index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+          <header class="listHeading">Pricing</header>
+          <ul>
+            <li>1: $1.99</li>
+            <li>2-50: $1.49</li>
+            <li>51+: $0.99</li>
+          </ul>
+        </section>
+
+      </article> 
+</div>
```
Comment (C-Stein) [inline index.html]: the closing div tag should follow the same indentation as the opening div tag (it should be inside the body)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+</body>
```
Comment (C-Stein) [inline index.html]: and body should be indented within the html

File: index.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html lang="en">
+<head>
```
Comment (C-Stein) [inline index.html]: both head and body tags should be indented since they're inside the html

**Comment by C-Stein** [comment]
Nice work over all. I especially got a kick out of the cards as your product! There are a few small changes needed in the html, but the css looks great. let me know when the html changes are done and I can check it one last time before you merge. Good Job!

**Comment by tonidew** [comment]
Thank you so much for all your help. 
And thank you for the quick review. 
Toni

**Comment by C-Stein** [comment]
Great job! Go ahead and merge :+1:

**Pull request merged**